mockgcp/common/httptogrpc/service.go

```diff
@@ -54,49 +110,67 @@ func (m *grpcMux) AddService(client any) {
 	}
```

Comment:
The logic `!goTypeMethodNames[string(method.Name())]` assumes a strict 1:1 mapping where every method in the Proto Service MUST exist in the Go client.

Is it guaranteed that the Go client (interface/struct) has exactly the same methods? Sometimes generated clients might have helper methods or the proto might have methods not yet generated/supported?

Just double checking if this strictness is intended.

Reply:
We have to match go methods to proto methods.  Strict works.  We should ask the agent to send a PR to make it better :-)